File: Facturador/app/src/main/java/com/example/facturador/Activity/Model/Equipo.java
package com.example.facturador.Activity.Model;

import java.io.Serializable;

public class Equipo implements Serializable {

    private int idequipo;
    private String serie;
    private String modelo;
    private int idestatus;
    private int idbanco;
    private int id_base;

    public Equipo() {

    }

    public int getId_base() {
        return id_base;
    }

    public void setId_base(int id_base) {
        this.id_base = id_base;
    }

    public Equipo(int idequipo, String serie, String modelo, int idestatus, int idbanco) {
        this.idequipo = idequipo;
        this.serie = serie;
        this.modelo = modelo;
        this.idestatus = idestatus;
        this.idbanco = idbanco;
    }

    public int getIdequipo() {
        return idequipo;
    }

    public void setIdequipo(int idequipo) {
        this.idequipo = idequipo;
    }

    public String getSerie() {
        return serie;
    }

    public void setSerie(String serie) {
        this.serie = serie;
    }

    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public int getIdestatus() {
        return idestatus;
    }

    public void setIdestatus(int idestatus) {
        this.idestatus = idestatus;
    }

    public int getIdbanco() {
        return idbanco;
    }

    public void setIdbanco(int idbanco) {
        this.idbanco = idbanco;
    }
}

File: webservices/CORE/Equipo.php
<?php


class Equipo
{

    public $idequipo,$serie,$modelo,$idestatus,$idbannco;

    /**
     * Equipo constructor.
     * @param $idequipo
     * @param $serie
     * @param $modelo
     * @param $idestatus
     * @param $idbannco
     */
    public function __construct($idequipo, $serie, $modelo, $idestatus, $idbannco)
    {
        $this->idequipo = $idequipo;
        $this->serie = $serie;
        $this->modelo = $modelo;
        $this->idestatus = $idestatus;
        $this->idbannco = $idbannco;
    }

    /**
     * @return mixed
     */
    public function getIdequipo()
    {
        return $this->idequipo;
    }

    /**
     * @param mixed $idequipo
     */
    public function setIdequipo($idequipo)
    {
        $this->idequipo = $idequipo;
    }

    /**
     * @return mixed
     */
    public function getSerie()
    {
        return $this->serie;
    }

    /**
     * @param mixed $serie
     */
    public function setSerie($serie)
    {
        $this->serie = $serie;
    }

    /**
     * @return mixed
     */
    public function getModelo()
    {
        return $this->modelo;
    }

    /**
     * @param mixed $modelo
     */
    public function setModelo($modelo)
    {
        $this->modelo = $modelo;
    }

    /**
     * @return mixed
     */
    public function getIdestatus()
    {
        return $this->idestatus;
    }

    /**
     * @param mixed $idestatus
     */
    public function setIdestatus($idestatus)
    {
        $this->idestatus = $idestatus;
    }

    /**
     * @return mixed
     */
    public function getIdbannco()
    {
        return $this->idbannco;
    }

    /**
     * @param mixed $idbannco
     */
    public function setIdbannco($idbannco)
    {
        $this->idbannco = $idbannco;
    }


}